runtime/src/main/java/io/quarkiverse/ironjacamar/runtime/IronJacamarRuntimeConfig.java

```diff
@@ -30,6 +30,16 @@
 @ConfigRoot(phase = ConfigPhase.RUN_TIME)
 public interface IronJacamarRuntimeConfig {
 
+    /**
+     * The maximum amount of time the worker thread can be blocked.
+     * If not specified it assumes the same value as defined by the <code>quarkus.vertx.max-worker-execute-time</code>
+     * configuration.
+     *
+     * @return the maximum amount of time the worker thread can be blocked
+     */
+    @WithDefault("${quarkus.vertx.max-worker-execute-time:60}")
```

Comment:
"PT60S" ?

Reply:
It is 60S by default, I checked and copied from https://github.com/quarkusio/quarkus/blob/main/extensions/vertx/runtime/src/main/java/io/quarkus/vertx/core/runtime/config/VertxConfiguration.java#L54